open_event/static/js/jquery/jquery.codezero.js

```diff
@@ -0,0 +1,221 @@
+/**
+ * Created by Niranjan on 29-May-16.
+ */
+
+/**
+ * An extended jQuery method to call a callback on an input change
+ */
+jQuery.fn.extend({
+    valueChange: function (callback) {
+        return this.each(function () {
+            var elem = $(this);
+            elem.data('oldVal', elem.val());
+            elem.bind("propertychange change click keyup input paste", function (event) {
+                if (elem.data('oldVal') != elem.val()) {
+                    elem.data('oldVal', elem.val());
+                    callback(elem.val(), event);
+                }
+            });
+        });
+    }
+});
+
+/**
+ * Cache the results of a RegExp match.
+ * @type {Function}
+ */
+var cachedFuzzyMatch = _.memoize(function (pattern) {
+    return new RegExp(pattern.split("").reduce(function (a, b) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2674855960/issues/source?bid=3233864&fileBranchId=3324174#l28)